.travis/push-javadoc-to-gh-pages.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [[ "$TRAVIS_REPO_SLUG" == "adorsys/keystore-management"
```

Comment:
Why not to return immediately if not matches

Reply:
done